addons/stock/models/stock_move.py

```diff
@@ -2203,3 +2203,46 @@ def _convert_string_into_field_data(self, string, options):
         if regex_findall(r'^([0-9]+\.?[0-9]*|\.[0-9]+)$', string):  # Number => Quantity.
             return {'quantity': float(string)}
         return False
+
+    def _filter_move_by_availability(self, operator, value, _get_comparison_date):
```

Comment:
I think it would be better to rename this method since it doesn't return the moves anymore.
`_is_expected_state`, `_match_searched_availability` or something like that ?